src/scicookie/{{cookiecutter.project_slug}}/scripts/check-gitlab-ci.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Replace with the actual GitLab project ID and GitLab API URL
+PROJECT_ID="{{ cookiecutter.git_username }}%2F{{ cookiecutter.project_slug }}"
```

Comment:
this code will not be handled by scicookie, instead this will be used as part of the smoke tests.
so use something fixed for now .. you can point to your own repo on gitlab for the test .. and I can provide a new repo on osl gitlab when this PR is ready